bitnami/contour/Chart.yaml

```diff
@@ -27,4 +27,4 @@ sources:
   - https://github.com/envoyproxy/envoy
   - https://github.com/bitnami/bitnami-docker-contour
   - https://projectcontour.io
-version: 4.3.10
+version: 4.4.10
```

Comment:
@marcosbc https://github.com/bitnami/charts/pull/6907#issuecomment-880763156 said that:

> I think we may need to increase the major version or chart upgrades will not work

Did you test the upgrade and works?